### PR TITLE
added a blog

### PR DESCRIPTION
**Pull request by Kylin93CN**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

File: _posts/2020-05-17-use-wechaty-manage-ur-customers.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: "使用Wechaty管理你的顾客"
+author: Kylin93CN
+date: '2020-05-17 19:20:50 +0800'
+categories: project
+tags:
+  - wechaty
+  - wechaty-puppet-macpro
```
Comment (lijiarui) [inline _posts/2020-05-17-use-wechaty-manage-ur-customers.md]: Please add a teaser here to make the blog more beautiful.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request, thanks!

**Comment by lijiarui** [comment]
@Kylin93CN Thanks for your post, please sign the CLA and change as I request, then I can merge this PR.

File: _posts/2020-05-17-use-wechaty-manage-ur-customers.md (hunk 1 above)
Comment (Kylin93CN) [reply _posts/2020-05-17-use-wechaty-manage-ur-customers.md]: done.

File: _posts/2020-05-17-use-wechaty-manage-ur-customers.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: "使用Wechaty管理你的顾客"
+author: Kylin93CN
+date: '2020-05-17 19:20:50 +0800'
+categories: XueMiEr-wx-bot
```
Comment (huan) [inline _posts/2020-05-17-use-wechaty-manage-ur-customers.md]: I can not understand what this category is.

File: _posts/2020-05-17-use-wechaty-manage-ur-customers.md (hunk 1 above)
Comment (lijiarui) [reply _posts/2020-05-17-use-wechaty-manage-ur-customers.md]: where is the teaser？

**Comment by lijiarui** [comment]
@Kylin93CN Please add a teaser(image) for this blog.

**Comment by lijiarui** [comment]
Congratulations! 🎉 🎉 🎉

You become a wechaty contributor successfully! 👏 👏 👏

Please:

- [ ]  Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room
- [ ]  Join wechaty org by clicking https://github.com/wechaty

**Review by lijiarui — APPROVED** [review]

**Pull request merged**